### PR TITLE
HPCC-13355 Remote copy group target display

### PR DESCRIPTION
**Pull request by miguelvazq**
Remote copy should only display thor,hthor,roxie as options within the group dropdown.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

### CODE REVIEW

**Comment by miguelvazq** [comment]
@wangkx Please review.

**Comment by wangkx** [comment]
@miguelvazq the code looks good. But, would you rename the method to something like: loadClusterGroups()?

**Comment by miguelvazq** [comment]
Can do. Doesn't effect other files that use this call as long as they continue using 'Groups: true'

**Comment by miguelvazq** [comment]
@ghalliday Ready to be merged when ready.

**Comment by ghalliday** [comment]
The change looks good. @jakesmith just wanted to check the change was correct (since it it for 5.2.2 and I am not familiar enough with the issue.)

File: esp/src/eclwatch/TargetSelectWidget.js [hunk 1]
```diff
@@ -170,17 +170,23 @@ define([
...
                     if (lang.exists("TpGroupQueryResponse.TpGroups.TpGroup", response)) {
                         var targetData = response.TpGroupQueryResponse.TpGroups.TpGroup;
                         for (var i = 0; i < targetData.length; ++i) {
-                            context.options.push({
-                                label: targetData[i].Name,
-                                value: targetData[i].Name
-                            });
+                            switch(targetData[i].Kind) {
+                                case "Thor":
+                                case "hthor":
```
Comment (richardkchapman) [inline esp/src/eclwatch/TargetSelectWidget.js]: Why is hthor all lower case while the others are capitalized? Is this comparison case-sensitive?

Comment (miguelvazq) [inline esp/src/eclwatch/TargetSelectWidget.js]: I verified with Kevin that the response is returning lowercase string htor this is how the response is returning the values. @wangkx can you verify?

Comment (wangkx) [inline esp/src/eclwatch/TargetSelectWidget.js]: It is a lowercase string. Both the "Thor" and the "hthor" are read from dali.

Comment (richardkchapman) [inline esp/src/eclwatch/TargetSelectWidget.js]: @jakesmith Any reason for this inconsistency?

Comment (jakesmith) [inline esp/src/eclwatch/TargetSelectWidget.js]: No good reason afaics.. just legacy constant strings used in the creation of Group(s) during Dali startup from the cluster topologies in the Environment.

**Comment by jakesmith** [comment]
Looks okay otherwise afaics.

**Pull request merged**